src/utils/inject-stylesheet.js

```diff
@@ -7,13 +11,56 @@
  *
  * @returns {void}
  */
-export function injectStylesheet(endpoint, env, version, uid, cid) {
-	const queryParams = (cid) ? '?previewcid=' + cid : '';
+export function injectStylesheetWithAutoReplace(endpoint, env, version, uid, cid, attempt = 0) {
+	injectStylesheet(endpoint, env, version, uid, cid)
+	const url = getStyleSheetUrl(endpoint, env, version, uid, cid);
+	const cssFile = getCSSFile(url);
+
+	if (cssFile && window.MutationObserver) {
+		const observer = new window.MutationObserver(function (mutationList) {
+			mutationList.forEach(function (mutation) {
+				mutation.removedNodes.forEach(function (node) {
+					if (node.nodeName === 'LINK' && node.href === url && attempt < MAX_RETRIES) {
```

Comment:
Should a separate warning go out if we hit the max retries?

Reply:
yep